src/Symfony/Component/Form/Extension/Core/View/ChoiceView.php

```diff
@@ -32,7 +51,9 @@ class ChoiceView extends BaseChoiceView
      */
     public function __construct($data, $value, $label)
     {
-        parent::__construct($label, $value, $data);
+        $this->data = $data;
+        $this->value = $value;
+        $this->label = $label;
 
         trigger_error('The '.__CLASS__.' class is deprecated since version 2.7 and will be removed in 3.0. Use Symfony\Component\Form\ChoiceList\View\ChoiceView instead.', E_USER_DEPRECATED);
```

Comment:
@stof no it should not because the new class does not call the parent class at all